static/js/components/material/Drawer.js

```diff
@@ -27,6 +27,14 @@ class Drawer extends React.Component {
     this.drawer = new MDCTemporaryDrawer(this.drawerRoot);
     this.drawer.listen('MDCTemporaryDrawer:close', onDrawerClose);
     this.updateRequirements();
+
+    let item = document.querySelector('#collapse_item');
+    if (item) {
+      item.addEventListener('click', (evt: MouseEvent) => {
+        evt.preventDefault();
+        onDrawerClose();
+      }, false);
+    }
```

Comment:
Can you add a comment explaining that this is a necessary hack to get around MDC limitations?

Reply:
sure